scripts/yocto-build-and-test.sh

```diff
@@ -126,16 +125,12 @@ PREFERRED_VERSION_pn-mender-binary-delta = "$mender_binary_delta_version"
 EOF
 
     local mender_monitor_version=$(tar -Oxf $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz ./mender-monitor/.version | egrep -o '[0-9]+\.[0-9]+\.[0-9b]+(-build[0-9]+)?')
-    for i in $WORKSPACE/stage-artifacts/mender-monitor-*.tar.gz; do
-        tar -C $WORKSPACE/stage-artifacts -xvzf $i
-    done
     if [ -z "$mender_monitor_version" ]; then
         mender_monitor_version="master-git"
```

Comment:
Note to self. This should now be:
```suggestion
        mender_monitor_version="master-git%"
```